route-rs-runtime/src/lib.rs

```diff
@@ -23,6 +23,8 @@ pub mod link;
 /// program takes the user provided graph, and generates a pipeline. It connects input and output channels to the pipe, so that it can be
 /// hooked up to packet sources and sinks, and drops the pipeline into the runtime. Generally can be abstracted to the concept, router.
 pub mod pipeline;
+pub mod packet;
```

Comment:
I think this could work better as a standalone crate -- that way I could use it in `afpacket` tests, for instance.

Reply:
I want to agree, would be nice to version the library too, so I can tell users it is 0.1 lol.  Will create a patch to move the files.